crates/types/src/data.rs

```diff
@@ -831,12 +818,12 @@ impl<TYPES: NodeType> LeafType for SequencingLeaf<TYPES> {
         self.parent_commitment
     }
 
-    fn get_deltas(&self) -> Self::DeltasType {
-        self.deltas.clone()
+    fn get_block_header(&self) -> <Self::NodeType as NodeType>::BlockHeader {
```

Comment:
Would it be better to return a reference here, allowing the caller to clone it if desired? It does commit us to having the header as a field of the leaf type, but I think that's probably a fair assumption